sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -416,7 +417,7 @@ export class EducationProgramOfferingControllerService {
       programId: program.id,
       programName: program.name,
       programDescription: program.description,
-      programCredential: program.credentialType,
+      programCredential: credentialTypeToDisplay(program.credentialType),
```

Comment:
Is there any issue if we follow the same approach from the `EducationProgramAPIOutDTO`

https://github.com/bcgov/SIMS/blob/7c7f20cf45757b5f6e3021b4e60cf2e184b815e1/sources/packages/web/src/services/http/dto/EducationProgram.dto.ts#L17-L18